### PR TITLE
Increase WebcompatExceptionsService in Release to 100%

### PR DESCRIPTION
**Pull request by arthuredelstein**

### CODE REVIEW

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers 👍

**Comment by goodov** [comment]
@arthuredelstein please enable the feature by default in brave-core. Ideally we should always do this before rolling the feature to Release channel.

**Pull request merged**
